src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.PersistenceEF/Utils/PreMigrationScripts/01-Main/area.sql

```diff
@@ -0,0 +1,74 @@
+-- AREA
+alter table dbo.area drop constraint uc_area_name;
+
+alter table dbo.area alter column audit_changedby drop not null;
+alter table dbo.area alter column audit_changedbysystem drop not null;
+alter table dbo.area alter column audit_changeoperation drop not null;
+alter table dbo.area alter column audit_validfrom drop default;
+
+ALTER TABLE dbo.area RENAME CONSTRAINT fk_area_group_areagroup TO fk_area_areagroup_groupid;
+
+ALTER INDEX dbo.uc_area_name_idx RENAME TO ix_area_name;
+ALTER INDEX dbo.fk_area_groupid_areagroup_idx RENAME TO ix_area_groupid;
+
+ALTER TRIGGER area_audit_update ON dbo.area RENAME TO audit_area_update_trg;
+ALTER TRIGGER area_audit_delete ON dbo.area RENAME TO audit_area_delete_trg;
+
+drop trigger area_meta on dbo.area;
+
+create or replace function dbo.audit_area_insert_fn() returns trigger
+    language plpgsql
+as
+$$
+BEGIN
+NEW.audit_validfrom := now();
+RETURN NEW;
+END;
+$$;
+
+create trigger audit_area_insert_trg
```

Comment:
Jeg endte opp med å legge på `create or replace` på alle disse trigger opprettelsene også ved lokal kjøring, da de også feilet ofte for meg sammen med funksjonen over som jeg ser du har oppdatert med dette